client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -1,6 +1,6 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
```

Comment:
Could you add the annotation as describe in [README](https://github.com/kubernetes-csi/external-snapshotter/blob/master/client/hack/README.md)
kind: CustomResourceDefinition
metadata:
  annotations:
    controller-gen.kubebuilder.io/version: v0.3.0
    `api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/139"`
Please go through the [README](https://github.com/kubernetes-csi/external-snapshotter/blob/master/client/hack/README.md) for details
Thanks

Reply:
@Kartik494 api-approved annotation already exists, they are removed.

Reply:
@prateekpandey14 I guess what @Kartik494 was asking is to update it to the pull request number to 367 (this one)?

Reply:
Hi @yuxiangqian, 

My understanding (by reading this https://github.com/kubernetes/enhancements/pull/1111)  is that "api-approved" annotation should be the PR which went through API reviews and was approved by the API reviewers.  So it should still be the original PR when snapshot API was promoted to Beta: "https://github.com/kubernetes-csi/external-snapshotter/pull/139"

I believe this annotation was removed automatically when @prateekpandey14 ran controller-gen. So @Kartik494 asked him to add it back manually.

Reply:
ok, I thought the ask was to update the pull request. if thats the case, never mind.